### ISSUE TITLE
Indentation used in Latex for \items

### ISSUE THREAD

**Issue by burblebee** [cplusplus/draft#3127]
Our guidelines[https://github.com/cplusplus/draft/wiki/Specification-Style-Guidelines?]()  say to use 2 space indentation for \items, which I find to be very helpful: it's clear to see where the bullets start and end, and when the bullets are nested.  When we edit old text which doesn't use this indentation however, we tend to stick with the old style for consistency.

Can we do a global whitespace change to fix the old formatting style so that we can follow the guidelines when applying edits?

**Comment by Eelis** [comment]
(#1899 also concerns a global whitespace policy change involving newlines.)

**Comment by jensmaurer** [comment]
Editorial teleconference: No consensus.

**Comment by jensmaurer** [comment]
See also #2110.

**Comment by jensmaurer** [comment]
For long \item, add an empty line in front and after.
Do not indent \item. Break after \item unless all of the bullets are very short.
Keep it as rough guidance, don't mass-update the source code.
Update the formatting wiki write-up accordingly.

**Comment by johelegp** [comment]
> Update the formatting wiki write-up accordingly.

Please, do.

**Comment by burblebee** [comment]
> > Update the formatting wiki write-up accordingly.
> > 
> Please, do.

Yes please!  It's not clear to me what is expected for the long items - fixing the wiki would make this clear.  From "For long \item, add an empty line in front and after", we'll have empty lines after the `\begin{itemize}` and before the `\end{itemize}`, often resulting in a `\begin{itemize}` or `\end{itemize}` statement having an empty line both before and after it which seems odd, is that what we want?  Perhaps we should instead have an empty line before the `\begin{itemize}` (except when starting a paragraph) and after the `\end{itemize}`?  Example:
```
\pnum
Blah blah blah...
Blah blah:

\begin{itemize}
\item
A multi-line text.

\item
Another multi-line text.
\end{itemize}

\pnum
Blah blah blah...
```

**Comment by Eelis** [comment]
> Perhaps we should instead have an empty line before the \begin{itemize} (except when starting a paragraph) and after the \end{itemize}

When the list is part of a sentence, inserting empty lines around the list breaks the sentence, which breaks sentence linking on eel.is/c++draft. See https://github.com/cplusplus/draft/issues/1899
